common/persistence/nosql/nosqlplugin/common.go

```diff
@@ -31,8 +31,8 @@ func getCadencePackageDir() (string, error) {
 	if err != nil {
 		panic(err)
 	}
-	cadenceIndex := strings.LastIndex(cadencePackageDir, "/cadence/")
-	cadencePackageDir = cadencePackageDir[:cadenceIndex+len("/cadence/")]
+	cadenceIndex := strings.LastIndex(cadencePackageDir, "cadence/")
+	cadencePackageDir = cadencePackageDir[:cadenceIndex+len("cadence/")]
```

Comment:
was this failing somewhere, or just felt like removing `/` prefixes?  seems fine either way since it doesn't change the behavior, just odd to see it change.

Reply:
My local folder is called my-cadence :D and it failed. I've tried switching to finding git root, but apparently our CI is running on windows and has some issues with safe-folder related stuff. So I just fixed the `/` part.

Reply:
our ci is definitely on linux, so it's not that at least.  either way this seems totally fine tho.

we could eliminate this entirely if we used `//go:embed` for the indexes, but that's a separate thing.

Reply:
Well, from the errors like from 
```git rev-parse --show-toplevel```
that I've got when using https://github.com/Integralist/go-findroot/blob/master/find/find.go#L36
```fatal: detected dubious ownership of /cadence```
means that we are running on windows with WSL.